app/controllers/admin/suppliers/supplier_users_controller.rb

```diff
@@ -67,7 +67,7 @@ def create
         @supplier_user_form.save!
         session.delete(:supplier_user_form)
 
-        set_success_message(heading: "User added", content: "They have been sent an email to sign in")
+        set_success_message(heading: "User added", content: "")
```

Comment:
Can we set content to be empty by default in the method, and not pass empty content?

Reply:
Is this temporary? If not, could I suggest `heading: "Success", content: "User added"`?

Reply:
Nope, Success is the title

Reply:
![image](https://user-images.githubusercontent.com/40488007/111773539-35977880-88a6-11eb-81c7-20bf040b35ec.png)
Double success would be weird